hummingbot/core/pubsub.pyx

```diff
@@ -65,6 +65,7 @@ cdef class PubSub:
         return self.c_get_listeners(event_tag.value)
 
     def trigger_event(self, event_tag: Enum, message: any):
+        print(f"Trigger_event type {event_tag} message {message}")
```

Comment:
I think this addition should be removed as well.

Reply:
You are right @CrimsonJacket, this has to be removed. I will do that in my next branch (since this has already been merged). Thank you for noticing this.